src/components/postFooter/GifContainer.jsx

```diff
@@ -14,7 +14,7 @@ const GifContainer = ({ displayTenor, removeTenor, handleSelectedGif }) => {
   const apikey = "AIzaSyCa4-1xZzD8bFenw_A53rzELEmXXSqlQAM";
   const clientkey = "my_test_app";
 
-  const searchResults = document.querySelector(".searchResults");
+  const searchResults = window.document.querySelector(".searchResults");
```

Comment:
can you kindly assist me understand why the change from
`document.querySelector(".searchResults")` to
`window.document.querySelector(".searchResults");`
and the subsequent other changes beneath?

Reply:
this was an eslint no undefined error when I used just document but when I changed it to window.document the error disapeared

Reply:
ahh i see wh then! cool.  I think we can change the eslint configuration then so that  we maintain it as the document.querySelector.

Reply:
@Izima01